src/assets/icons/Add/Add.module.scss

```diff
@@ -0,0 +1,32 @@
+.circle {
+	position: relative;
+	width: 17px;
+	height: 17px;
+	border-radius: 50%;
+	border: 0.5px solid black;
+	color: black;
+	background-color: white;
+	transition: all 0.2s;
+	cursor: pointer;
+}
+
+.circle:hover {
+	color: white;
```

Comment:
**nitpick:** Let's create variables for colors, so you dont have to duplicate then everytime